Corona-Warn-App/src/test/java/de/rki/coronawarnapp/diagnosiskeys/download/CommonSyncToolTest.kt

```diff
@@ -160,3 +141,39 @@ abstract class CommonSyncToolTest : BaseIOTest() {
         }
     }
 }
+
+internal val String.loc get() = LocationCode(this)
+internal val String.day get() = LocalDate.parse(this)
+internal val String.hour get() = LocalTime.parse(this)
+
+fun createMockCachedKeyInfo(
+    dayIdentifier: LocalDate,
+    hourIdentifier: LocalTime?,
+    isComplete: Boolean,
+    location: LocationCode = "EUR".loc,
+): CachedKeyInfo {
```

Comment:
We already had a mocking function that could be reused in other tests, I've done some refactoring to do that.